syntax = "proto3";

package helloworld;

service LoggerService {
  rpc ContactServer (ContactRequest) returns (ContactResponse) {}
}

message ContactRequest {
  string um = 1;
  string dm = 2;
  string timestamp_sent = 3;
  string timestamp_actual = 4;
  string communication_type = 5;
}

message ContactResponse {
  string status = 1;  // Status message indicating success or failure
}
